### PR TITLE
Email attachment example

### PR DESCRIPTION
**Pull request by gewenyu99**
## What does this PR do?

(Provide a description of what this PR does and why it's needed.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [ ] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

**Review by eldadfux — APPROVED** [review]

File: app/controllers/api/messaging.php [hunk 1]
```diff
@@ -2596,7 +2596,7 @@
     ->param('targets', [], new ArrayList(new UID()), 'List of Targets IDs.', true)
     ->param('cc', [], new ArrayList(new UID()), 'Array of target IDs to be added as CC.', true)
     ->param('bcc', [], new ArrayList(new UID()), 'Array of target IDs to be added as BCC.', true)
-    ->param('attachments', [], new ArrayList(new CompoundUID()), 'Array of compound bucket IDs to file IDs to be attached to the email.', true)
+    ->param('attachments', [], new ArrayList(new CompoundUID()), 'Array of compound ID strings of bucket IDs and file IDs to be attached to the email. They should be formatted as <BUCKET_ID>:<FILE_ID>.', true)
```
Comment (stnguyen90) [inline app/controllers/api/messaging.php:2599]: probably need this for create and update push too...and why isn't this param in update email?? 😬

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

File: app/controllers/api/messaging.php (hunk 1 above)
Comment (eldadfux) [reply app/controllers/api/messaging.php:2599]: +1, @stnguyen90 can you take the lead on fixing this?

**Comment by 7777chaitanya** [comment]
Hey @eldadfux I was randomly checking the repository. I am curious, Why this PR doesn't have a related issue? I would like to know how this change was requested from the contributor by the lead? Did the team have an internal discussion?

**Comment by ItzNotABug** [comment]
> Hey @eldadfux I was randomly checking the repository. I am curious, Why this PR doesn't have a related issue? I would like to know how this change was requested from the contributor by the lead? Did the team have an internal discussion?

This was possibly discussed internally because the PR is targeted to `1.5`'s release which isn't publicly available yet.

File: app/controllers/api/messaging.php (hunk 1 above)
Comment (stnguyen90) [reply app/controllers/api/messaging.php:2599]: Created https://github.com/appwrite/appwrite/issues/7883 for me to work on

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Please also remove the package-lock.json from the PR 🙏🏼

**Review by stnguyen90 — APPROVED** [review]

**Pull request merged**